modules/configuration/src/test/java/org/apache/ignite/internal/configuration/ConfigurationChangerTest.java

```diff
@@ -113,13 +112,13 @@ public static class ThirdConfigurationSchema {
         public String strCfg;
     }
 
-    private static ConfigurationAsmGenerator cgen = new ConfigurationAsmGenerator();
+    private static ConfigurationTreeGenerator generator = new ConfigurationTreeGenerator(List.of(KEY, DefaultsConfiguration.KEY));
 
     private final TestConfigurationStorage storage = new TestConfigurationStorage(LOCAL);
 
     @AfterAll
-    public static void afterAll() {
-        cgen = null;
+    public static void afterAll() throws Exception {
```

Comment:
Does it have to throw an exception? This "close" method in generator simply nullifies the field, there's no way it ends with exception.